main.tf

```diff
@@ -106,7 +110,7 @@ resource "azurerm_role_definition" "lacework" {
 }
 
 resource "azurerm_role_assignment" "lacework" {
-  role_definition_id = azurerm_role_definition.lacework.id
+  role_definition_id = azurerm_role_definition.lacework.role_definition_resource_id
```

Comment:
Thanks for updating this, though, have you verified that this change works the same way in the `azurerm` version `2.26`?

We have a hard dependency to that version and if Microsoft has updated the provider and we can use the latest, then let us update this file as well please: https://github.com/lacework/terraform-azure-activity-log/blob/main/versions.tf#L6

Reply:
Oye, It looks like this change went into effect in `2.29` - I missed the boat that the docs say pin to version `2.26`, so let me revert this change.  Apologies! 🤦